### PR TITLE
Geometry changes for GE21- QC8 simulations

### PR DESCRIPTION
**Pull request by manuelrodriguezgiraldo**
the changes we talk about today

### CODE REVIEW

File: Geometry/data/gem21.xml [hunk 1]
```diff
@@ -427,10 +427,10 @@
     <rSolid name="GEMBox21L"/>
     <rMaterial name="materials:Air"/>
   </LogicalPart>
-  <!-- <LogicalPart name="GEMDisk21LN" category="unspecified">
+  <LogicalPart name="GEMDisk21LN" category="unspecified">
```
Comment (jshlee) [inline Geometry/data/gem21.xml:430]: why is this changed?

File: Geometry/test/testGeometryQC8GE21.py [hunk 2]
```diff
@@ -3,15 +3,15 @@
...
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
 process.load('gemsw.Geometry.GeometryQC8GE21_cff')
-#process.load('Configuration.StandardSequences.MagneticField_0T_cff')
-#process.load('Configuration.StandardSequences.Reconstruction_cff')
-#process.load('Configuration.StandardSequences.RecoSim_cff')
-#process.load('Configuration.StandardSequences.EndOfProcess_cff')
+process.load('Configuration.StandardSequences.MagneticField_0T_cff')
+process.load('Configuration.StandardSequences.Reconstruction_cff')
+process.load('Configuration.StandardSequences.RecoSim_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
```
Comment (jshlee) [inline Geometry/test/testGeometryQC8GE21.py:14]: why do you load these??

Comment (yeckang) [reply Geometry/test/testGeometryQC8GE21.py:14]: @manuelrodriguezgiraldo Could you comment out these lines like `testGeometryTestBeam.py`?
https://github.com/gem-sw/gemsw/blob/main/Geometry/test/testGeometryTestBeam.py

**Comment by jshlee** [comment]
what is the difference between teststand and teststand2?
I think it would be good to have teststandFront, teststandBack and teststandMix

File: Geometry/test/testGeometryQC8GE21.py (hunk 2 above)
Comment (manuelrodriguezgiraldo) [reply Geometry/test/testGeometryQC8GE21.py:14]: at one point I was not loading the geometry and I was checking if it worked and I forgot to remove this

**Comment by manuelrodriguezgiraldo** [comment]
what is the difference between teststand and teststand2?
I think it would be good to have teststandFront, teststandBack and teststandMix

I was looking at two different configurations of a single type of camera and in order not to be changing all the time on the same xlm I created another

File: Geometry/data/gem21.xml (hunk 1 above)
Comment (manuelrodriguezgiraldo) [reply Geometry/data/gem21.xml:430]: In the same process of checking what the differences were, I found that it was the only comment that was in the logic part and I removed it to see what happened and I forgot to return the comment, if you want I can remove that part of the code and make the pull request of new

File: Geometry/data/QC8GE21/teststand.xml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0"?>
 <DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../../DetectorDescription/Schema/DDLSchema.xsd">
     <ConstantsSection label="gem21.xml" eval="true">
-        <Constant name="layers"       value="5"/>
+        <Constant name="layers"       value="7"/>
```
Comment (jshlee) [inline Geometry/data/QC8GE21/teststand.xml:4]: why do you change this?
is this used?

File: Geometry/data/gem21.xml (hunk 1 above)
Comment (jshlee) [reply Geometry/data/gem21.xml:430]: this was commented out for a reason

File: Geometry/test/testGeometryQC8GE21.py (hunk 2 above)
Comment (jshlee) [reply Geometry/test/testGeometryQC8GE21.py:14]: remove please

**Comment by jshlee** [comment]
you only need to change teststand
remove all other changes

File: Geometry/data/QC8GE21/teststand.xml (hunk 3 above)
Comment (manuelrodriguezgiraldo) [reply Geometry/data/QC8GE21/teststand.xml:4]: It is not used, I only put it to always keep in mind that QC8 has 7 layers and not 5

File: Geometry/data/QC8GE21/gem21.xml [hunk 4]
```diff
@@ -424,9 +424,13 @@
 
 <LogicalPartSection label="gem21.xml">
   <LogicalPart name="GEMDisk21LP" category="unspecified">
-    <rSolid name="mf:ME2Ring"/>
+    <rSolid name="GEMBox21L"/>
```
Comment (jshlee) [inline Geometry/data/QC8GE21/gem21.xml:427]: why is this changed?

File: Geometry/data/gem21.xml [hunk 5]
```diff
@@ -424,13 +424,9 @@
 
 <LogicalPartSection label="gem21.xml">
   <LogicalPart name="GEMDisk21LP" category="unspecified">
-    <rSolid name="GEMBox21L"/>
+    <rSolid name="mf:ME2Ring"/>
```
Comment (jshlee) [inline Geometry/data/gem21.xml:427]: dont change this file, it is used by other stuff
